tests/run_tests.sh

```diff
@@ -356,6 +356,36 @@ check_test_results
 #EXIT_CODE=$?
 #EXPECTED_EXIT_CODE=0
 #check_test_results
+TEST=test_26
+echo -e "\033[1;33mSKIPPING\033[0m $TEST"
+
+
+# Test that "Returning error because of i/o error during file close" is fatal
```

Comment:
Q: Not certain I understand how the I/O error is simulated?

Reply:
I created a script in the test directory (`simulate_multx_error_test_27.pl`), which is supplied to the `--fastq-multx` option on line 368 below. That option is used to specify a non-default `fastq-multx` executable to be used in the run.

Reply:
I use the same option to simulate a non-0 exit from fastq-multx (`simulate_multx_non0_test_28.pl`) on line 382.

Reply:
Oh, I get it, you are just simulating/printing the error message, not somehow actually triggering an i/o error.